Security/Risk_management/Rapid_Risk_Assessment.mediawiki

```diff
@@ -0,0 +1,356 @@
+* Likelihood, Security provided by service. Don't spent much time there! These are "10, 000 feet approximations" and part of a larger risk calculation mechanism.
+
+== Guided process: Running your RRA in ~30 minutes ==
+
+This is a guided example of how to run an RRA.
+
+{| class="wikitable"
+|-
+! <span style="color:gray;">Time management and control</span>
+|-
+|
+You will be responsible for the time distribution and ensure 30 minutes are not exceeded if possible.
+
+This means that you will sometimes have to cut a discussion short. An incomplete RRA due to time being exceeded has no
+value. Do not hesistate to take control and enforce your own discussion topics. Examples of when you'll need to be
+assertive and regain control:
+
+* The discussion languishes (>1 minute) around "how to mitigate this very issue" or "we're doing X to ensure this never happens".
+* The discussion focuses on process instead of filling the form.
+* The discussion about how the service works takes forever (>15 minutes) and the owner has to lookup every single detail.
+
+A good tip is to '''reserve 50-60 minutes''' of RRA time in the calendar, and plan to run the RRA for only '''30
+minutes'''.
+This leaves you with some room for error, and handle services that weren't well understood by their owners.
+|}
+
+=== Getting started (pre-work) ===
+
+* Ensure no previous RRA exist, else re-use the previous RRA.
+* Create a copy of the [https://drive.google.com/open?id=160V89R-VdIe1AEHcT_sX89UToV2gv8mLshhoia2-6mM template] and move it to the correct directory (or your personal Google drive if you're testing the RRA process).
+* Invite 1 or 2 members related to the service with enough technical knowledge, ensure they will bring a diagram of data
+flows and have an understanding of the data being stored or processed by the service. You do not want more than 4 or 5 persons total as this will slow down the RRA. Most RRAs are run 1 on 1 (2 persons total).
+Make sure everyone invited has '''edit''' rights to the document, and has the document opened in front of them when the
+RRA starts.
+* If this is anyone's first RRA, ensure they understand what risk impacts are, and the standard risk levels we are using, as well giving them a short overview of what is going to happen:
+** Filling in header/meta-data information about the service.
+** Getting an idea of how the service is architected.
+** Recording all data the service may process/store and how sensitive it is.
+** Run through the risk table and figure out what happens if the data leak, is modified, is unavailable, etc - while
+assigning the standard risk levels.
+** Write down any recommendation for the service.
+
+{| class="wikitable"
+|-
+! <span style="color:red;">First time?</span>
+|-
+|
+If this is your first RRA, ensure that someone who has run RRAs previously is present to help you.
+It is good to have attended multiple RRAs before starting your own.
+|}
+
+=== Header (5-10 minutes) ===
```

Comment:
this should be prefilled if possible. 10 minutes on this is way too long! I'd say 1 to 3 minutes max.